node/src/manager/commands/assign.rs

```diff
@@ -109,3 +111,15 @@ pub fn pause_or_resume(
 
     Ok(())
 }
+
+pub fn restart(
+    primary: ConnectionPool,
+    sender: &NotificationSender,
+    search: &DeploymentSearch,
+    sleep: Duration,
+) -> Result<(), Error> {
+    pause_or_resume(primary.clone(), sender, search, true)?;
+    thread::sleep(sleep);
```

Comment:
Would be great to print out a message to console.. saying something like "Waiting for x seconds"

Actually might be good to have a "Pausing <Deployment ID>"

and a "resuming <Deployment ID> too

Reply:
Logs as of now:
```
pausing QmTGXDUdbYNuThTAUbeSA15BcQCRxX6qR1PyPN23e3JXFJ[210]
Operation completed
resuming QmTGXDUdbYNuThTAUbeSA15BcQCRxX6qR1PyPN23e3JXFJ[210]
Operation completed
```
I will add an entry for the sleeping duration.

Reply:
Perfect, I'll test it out